.github/workflows/coderabbit.yml

```
@@ -0,0 +1 @@
+language: ko
```

actionlint (1.7.7)

unexpected key "language" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)